public/canvas/canvas-customization.js

```diff
@@ -0,0 +1,864 @@
+      $helpMenuItem.after($academicPoliciesMenuItemResponsive);
+    }
+  });
+};
+
+addAcademicPoliciesResponsiveLink();
+
+/* MENTAL HEALTH RESOURCES RESPONSIVE LINK */
+
+  /**
+   * Add 'Mental Health Resources' above 'Help' in left-hand global navigation
+   */
+
+  var addMentalHealthResourcesResponsiveLink = function() {
+  waitUntilAvailable("div span ul li:contains('Inbox')", true, function($helpMenuItem) {
+    const isElementPresent = !!document.getElementById('global_nav_mental_health_resources_link_responsive')
+    if (!isElementPresent) {
+      var $mentalHealthResourcesMenuItemResponsive = $helpMenuItem.clone();
+      var $mentalHealthMenuTextElement = $mentalHealthResourcesMenuItemResponsive.find('span span span').first()
+      $mentalHealthMenuTextElement.text('Mental Health Resources');
+      $mentalHealthMenuTextElement.next().remove();
+      $mentalHealthResourcesMenuItemResponsive.find('a')
+        .attr('id', 'global_nav_mental_health_resources_link_responsive')
+        .attr('href', 'https://uhs.berkeley.edu/mental-health')
+        .attr('target', '_blank')
+        .attr('role', null)
+        .attr('data-track-category', null)
+        .attr('data-track-label', null);
+      // Add SVG source for https://fontawesome.com/icons/handshake-angle
+      $mentalHealthResourcesMenuItemResponsive.find('path')
+        .attr('transform', 'scale(3.8,3.8)')
+        .attr('d', 'M488 191.1h-152l.0001 51.86c.0001 37.66-27.08 72-64.55 75.77c-43.09 4.333-79.45-29.42-79.45-71.63V126.4l-24.51 14.73C123.2 167.8 96.04 215.7 96.04 267.5L16.04 313.8c-15.25 8.751-20.63 28.38-11.75 43.63l80 138.6c8.875 15.25 28.5 20.5 43.75 11.75l103.4-59.75h136.6c35.25 0 64-28.75 64-64c26.51 0 48-21.49 48-48V288h8c13.25 0 24-10.75 24-24l.0001-48C512 202.7 501.3 191.1 488 191.1zM635.7 154.5l-79.95-138.6c-8.875-15.25-28.5-20.5-43.75-11.75l-103.4 59.75h-62.57c-37.85 0-74.93 10.61-107.1 30.63C229.7 100.4 224 110.6 224 121.6l-.0004 126.4c0 22.13 17.88 40 40 40c22.13 0 40-17.88 40-40V159.1h184c30.93 0 56 25.07 56 56v28.5l80-46.25C639.3 189.4 644.5 169.8 635.7 154.5z');
+      $helpMenuItem.after($mentalHealthResourcesMenuItemResponsive);
+    }
+  });
+};
+
+addMentalHealthResourcesResponsiveLink();
+
+  /* E-GRADES EXPORT */
+
+  /**
+   * Add the 'E-Grades' export option to the Canvas Gradebook
+   */
+  var addEGrades = function() {
+    // Verify that the current context is the Gradebook tool
+    if (window.ENV && window.ENV.GRADEBOOK_OPTIONS && window.ENV.GRADEBOOK_OPTIONS.context_id) {
+      // Verify that the current course contains official course sections
+      var courseId = window.ENV.GRADEBOOK_OPTIONS.context_id;
+      apiRequest('/api/academics/canvas/egrade_export/is_official_course.json?canvas_course_id=' + courseId, function(officialCourseResponse) {
```

Comment:
@mlincoln - I'm not sure we have this API endpoint written yet. Check `canvas_site_controller.py`